tools/windows-node-installer/README.md

```diff
@@ -107,7 +107,7 @@ are run. In addition windows-node-installer.json and winworker-* files need to b
 
 To run the e2e change the directory to the `tools\windows-node-installer` of this repo and run the following commands:
 ```bash
-export ARTIFACT_DIRECTORY=$(pwd)
+export ARTIFACT_DIR=$(pwd)
```

Comment:
Why is this needed?

Reply:
in the code its referenced to as ARTIFACT_DIR whereas in readme its given as ARTIFACT_DIRECTORY

Reply:
Sure, but you should not fix that as part of this PR.